blueocean-personalization/src/main/js/redux/FavoritesStore.jsx

```diff
@@ -30,7 +30,9 @@ function clone(json) {
 
 const actionHandlers = {
     [ACTION_TYPES.SET_USER](state, { payload }) {
-        const user = new User(payload);
+        const user = payload instanceof Error ?
```

Comment:
so this has the side effect possibly that if there is a bug in fetching fav's, they may not just show up?